shremote.py

```diff
@@ -513,7 +522,17 @@ def safe_eval(st, label=''):
         log_error("Error evaluating expression '{}' due to [{}]".format(label, e))
         raise
 
+def try_eval(raw, label):
+    try:
+        evaled = safe_eval(raw, label)
+        if str(evaled) != str(raw): # Means that 'eval' did something
+            log_warn("Specifying evaluatable begin without $(...) is deprecated: {}".format(raw))
```

Comment:
This warning was only in place for the `begin` entry, because that was the only one where I was calling `eval()` automatically before. Is this currently throwing a warning every time that it is called?